src/gui/mainwindow.cpp

```diff
@@ -1352,6 +1359,12 @@ void MainWindow::on_actionTop_tool_bar_triggered()
     Preferences::instance()->setToolbarDisplayed(is_visible);
 }
 
+void MainWindow::on_actionShow_Status_bar_triggered() {
+    bool is_visible = static_cast<QAction*>(sender())->isChecked();
+    QMainWindow::statusBar()->setVisible(is_visible);
```

Comment:
The better way:
```c++
statusBar()->setVisible(is_visible);
```

In C++ we use fully qualified names to avoid ambiguity. Here it is not required.